data/projects.yml

```diff
@@ -424,3 +424,17 @@ project_list:
     github_user:
       - geealbers
     note: "Quire was used to present the findings of a study of online scholarly catalogues, commissioned by the Art Institute of Chicago, the J. Paul Getty Museum, the National Gallery of Art, and the Philadelphia Museum of Art. The study's goal was to better understand the digital publishing landscape for museums as well as the catalogue readers' experience and needs."
+
+  - id: "materia_vol_1"
+    src: /screenshots/_materia_vol_1.jpg
+    alt: "Materia: Journal of Technical Art History Vol. 1, Issue 1"
+    type: Journal
+    publisher: Materia: Journal of Technical Art History
```

Comment:
@audreywarne The deploy for this PR is breaking (that's why you see all the checks failed notifications). I believe it's the : in the publisher name. Please wrap this in quotes and double-check by running preview. Let me know if that fixes the error.